dataservice/api/participant/models.py

```diff
@@ -27,3 +28,7 @@ class Participant(db.Model, Base):
     demographic = db.relationship(Demographic, backref='participant',
                                   uselist=False, cascade="all, delete-orphan",
                                   lazy=True)
+    phenotypes = db.relationship(Phenotype,
+                                 cascade="all, delete-orphan",
+                                 backref=db.backref('participants',
+                                                    lazy=True))
```

Comment:
Can we set `passive_deletes=True` here to use the db to delete (much faster)

Reply:
👍